docs/pipelines/edit-pipeline.md

```diff
@@ -20,8 +20,7 @@ required_permissions:
 1. On the upper-right corner of the applications details view click `SETTINGS`
 1. Use the drop-down menu to choose `PIPELINES`
 1. In the list of pipelines find the pipeline you want to edit and click `DETAILS`
-1. On the upper-right corner of the key details click `...`
-1. Use the drop-down menu to choose `EDIT`
+1. On the upper-right corner of the key details click `EDIT`
```

Comment:
@guipontes7 could you update this PR applying the changes below ? I believe this is more aligned with the article objective of editing a pipeline

```suggestion
1. On the upper-right corner of the pipeline details view, click `EDIT`
```

Reply:
Just clarifying, this was a bug that i noticed while reviewing your PR but not caused by your changes in any way.

Reply:
File updated!